x-pack/plugins/security_solution/public/management/components/policy_response/policy_response_artifact_item.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { truncate } from 'lodash';
+import styled from 'styled-components';
+import { EuiCopy, EuiFlexGroup, EuiFlexItem, EuiIcon, EuiText, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import type { HostPolicyResponseAppliedArtifact } from '../../../../common/endpoint/types';
+
+const StyledArtifactName = styled(EuiText)`
+  white-space: nowrap;
+  line-height: inherit;
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+const StyledShaValue = styled(EuiText)`
+  width: 80px;
```

Comment:
instead of absolute units, can we use sizes from EUI? like this example
```
    padding-top: ${({ theme }) => theme.eui.euiSizeS};
```

Reply:
Switched to the `theme` where applicable.

d3e7f1de7435d312af2b4b87866ee8787526e9da